examples/templates/vue/README.md

```diff
@@ -0,0 +1,61 @@
+# {{ YOUR TITLE GOES HERE }}
+
+## Description
+
+{{ YOUR DESCRIPTION GOES HERE }}
+
+## Live preview
+
+You can preview this template on CodeSandbox  under the URL: https://githubbox.com/handsontable/code-snippets/tree/master/examples/templates/vue
```

Comment:
Since this is a template we should place a wildcard so the dev will notice it has to be updated:

```suggestion
You can preview this template on CodeSandbox  under the URL: https://githubbox.com/handsontable/code-snippets/tree/master/{{ PATH TO THE EXAMPLE }}
```

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/4cedf4a98e61e6ca3bd794aacadbac53b0f563fa